config/tests/jobs/jobs_test.go

```diff
@@ -706,7 +706,7 @@ func checkKubekinsPresets(jobName string, spec *coreapi.PodSpec, labels map[stri
 			}
 		}
 
-		if scenario == "kubenetes_e2e" {
+		if scenario == "kubernetes_e2e" {
```

Comment:
this one looks like it may have a counterpart somewhere?

Reply:
Yes, we have file https://github.com/kubernetes/test-infra/blob/master/scenarios/kubernetes_e2e.py

Reply:
Does that mean this job works incorrectly now until the typo is fixed, or that something relies on that fix? It looks like the former...